TACC/Core-Styles
# fix(c-card): image card padding not unset

## Overview

Make sure card padding for image cards can be removed.

## Changes

- **changed** specificity of a selector

## Testing

1. Visit http://localhost:3000/components/preview/c-card--images.
2. Scroll to "Plain Card Images".
3. Verify no padding around images.

## UI

| Before | After |
| - | - |
| <img width="920" alt="before" src="https://github.com/user-attachments/assets/07115cef-fb51-4adc-bf0f-56b510aa4bd9"> | <img width="920" alt="after" src="https://github.com/user-attachments/assets/28e4347b-96e4-4fe9-b366-ef6a8b7adee5"> |
